tests/integration/kubernetes/gha-run.sh

```diff
@@ -33,6 +33,7 @@ HTTPS_PROXY="${HTTPS_PROXY:-${https_proxy:-}}"
 NO_PROXY="${NO_PROXY:-${no_proxy:-}}"
 export AUTO_GENERATE_POLICY="${AUTO_GENERATE_POLICY:-no}"
 export TEST_CLUSTER_NAMESPACE="${TEST_CLUSTER_NAMESPACE:-kata-containers-k8s-tests}"
+export GENPOLICY_PULL_METHOD="${GENPOLICY_PULL_METHOD:-oci-distribution}"
```

Comment:
Oop, I missed this. Why? :) It doesn't make much difference to me since nothing is using that variable yet. Maybe better to leave it and "reserve" the env variable for us since we'll use it soon?

Reply:
The goal of this PR is just to add a variable that cannot be added in the PR that will use it.

If it's not used here yet, then it's not tested yet, so it's better to avoid adding it untested.

But I agree this is not a big deal.

Reply:
Agree with Dan on principle but also that it's not a big deal for this time. Just always good to be mindful of your reviewers' time (e.g. I had to read through this thread for a change that has no consequence) 🙂

Reply:
Removed :)